html/cross-origin-embedder-policy/shared-workers.html

```diff
@@ -0,0 +1,134 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<title>COEP - policy derivation for Shared Workers</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/get-host-info.sub.js"></script>
+<body>
+<p>Verify the Cross-Origin Embedder Policy for Shared Workers by performing a
+cross-domain "fetch" request for a resource that does not specify a COEP. Only
+Shared Workers with the default COEP should be able to successfully perform
+this operation.</p>
+<script>
+'use strict';
+const testUrl = get_host_info().HTTPS_REMOTE_ORIGIN +
+  '/html/cross-origin-embedder-policy/resources/empty-coep.py';
+const workerHttpUrl = get_host_info().HTTPS_ORIGIN +
+  '/html/cross-origin-embedder-policy/resources/shared-worker-fetch.js.py';
+let workerBlobUrl;
+let workerDataUrl;
+
+promise_setup(() => {
+  return fetch(workerHttpUrl)
+    .then((response) => response.text())
+    .then((text) => {
+      workerDataUrl = 'data:text/javascript;base64,' + btoa(text);
+      workerBlobUrl = URL.createObjectURL(
+        new Blob([text], { 'Content-Type': 'text/javascript' })
+      );
+    });
+});
+
+/**
+ * Create a Shared Worker within an iframe
+ *
+ * @param {object} t - a testharness.js subtest instance (used to reset global
+ *                     state)
+ * @param {string} ownerCoep - the Cross-Origin Embedder Policy of the iframe
+ * @param {string} workerUrl - the URL from which the Shared Worker should be
+ *                             created
+ */
+function create(t, ownerCoep, workerUrl) {
+  const iframe = document.createElement('iframe');
+  iframe.src = 'resources/empty-coep.py' +
+    (ownerCoep ? '?value=' + ownerCoep : '');
+
+  return new Promise((resolve, reject) => {
+      document.body.appendChild(iframe);
+      t.add_cleanup(() => iframe.remove());
+      iframe.onload = () => resolve(iframe);
+      iframe.onerror = () => {
```

Comment:
I believe iframes currently always fire a `load` event, even in the case of an error. So this is essentially dead code and can be removed.